### ISSUE TITLE
typescript support for images in env.d.ts

### ISSUE THREAD

**Issue by diracs-delta** [quasarframework/quasar#8114]
**Is your feature request related to a problem? Please describe.**
Basically, whenever I import images into my Vue SFCs, I do something like this

```
<template>
...
  <q-img
      :src="CoolImage"
   />
...
</template>

<script>
import CoolImage from 'src/assets/cool_image.jpg'
</script>
```

But Quasar's TS template doesn't add typings for images imported via webpack loaders. So every time I start a new Quasar project with TS, I have to add the following to my `env.d.ts` to get TS to recognize `CoolImage` as a string

```
// type declarations for images loaded via webpack
declare module '*.png' {
  const value: string
  export default value
}

declare module '*.jpg' {
  const value: string
  export default value
}

declare module '*.jpeg' {
  const value: string
  export default value
}
```

**Describe the solution you'd like**
I would like the above to ship in Quasar's templates by default when I enable TS support.
This doesn't look like that difficult of an addition, so I would also appreciate just being pointed towards which source files I could play around with to add this into the template.

**Describe alternatives you've considered**
Maybe I'm not supposed to use `import` statements in this case? This is always how I imported images in Quasar.

**Additional context**
i am on the quasar discord: senpie#1355

**Comment by IlCallo** [comment]
This is the first time I see someone importing images directly into JS/TS actually 🤔 
Quasar automatically embeds inline smaller images, and probably performs other optimizations, when it reads an image tag with `src="...."` pointing to the assets folder.
What's the advantage of loading them that way instead? Which loaders are you using? Are they shipped with Quasar by default?

**Comment by diracs-delta** [comment]
> What's the advantage of loading them that way instead?

I originally had wanted to use a custom pre-loader (`image-webpack-loader` to compress images) in the build step, but I've scrapped that idea simply because I couldn't get it working. Also, this is always how I imported images in other frameworks b/c other frameworks typically weren't smart enough to transpile image paths, so using importing images like this was necessary to generate inline base64 :smile: .

> Which loaders are you using? Are they shipped with Quasar by default?

I'm using the default Quasar webpack loader for images, which should be `url-loader`. However, this can't be the complete picture since `import CoolImage from 'src/assets/cool_image.jpg` returns `img/cool_image.jpg`, which is actually the path where image assets get compiled to when one runs `quasar build && cd dist/spa`. I believe this is because of babel, but I really haven't used enough babel to know what's going on.

Well, I guess I'll take advantage of Quasar's built-in features. Is there any place in docs that talks about how Quasar handles image paths?
